[
{"message":"file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/index.js:237\n        throw new Error(y, { cause: u });\n              ^\n","level":"FATAL","type":"PLAYGROUND","test":"jspi-boot","plugin":"comments-counter-in-feed","details":"file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/index.js:237\n        throw new Error(y, { cause: u });\n              ^\n","log":"/tmp/tmp.EZHgqVAi8t"}
,
{"message":"Function _load_textdomain_just_in_time was called <strong>incorrectly</strong>. Translation loading for the <code>cartnotify</code> domain was triggered too early. This is usually an indicator for some code in the plugin or theme running too early. Translations should be loaded at the <code>init</code> action or later. Please see <a href=\"https://developer.wordpress.org/advanced-administration/debug/debug-wordpress/\">Debugging in WordPress</a> for more information. (This message was added in version 6.7.0.) in /wordpress/wp-includes/functions.php on line 6121","level":"NOTICE","type":"PHP","test":"jspi-boot","plugin":"comments-counter-in-feed","details":"[30-May-2025 14:29:56 UTC] PHP Notice:  Function _load_textdomain_just_in_time was called <strong>incorrectly</strong>. Translation loading for the <code>cartnotify</code> domain was triggered too early. This is usually an indicator for some code in the plugin or theme running too early. Translations should be loaded at the <code>init</code> action or later. Please see <a href=\"https://developer.wordpress.org/advanced-administration/debug/debug-wordpress/\">Debugging in WordPress</a> for more information. (This message was added in version 6.7.0.) in /wordpress/wp-includes/functions.php on line 6121","log":"/logs/plugins/c/comments-counter-in-feed/error.json"}
,
{"message":"Function _load_textdomain_just_in_time was called <strong>incorrectly</strong>. Translation loading for the <code>cartnotify</code> domain was triggered too early. This is usually an indicator for some code in the plugin or theme running too early. Translations should be loaded at the <code>init</code> action or later. Please see <a href=\"https://developer.wordpress.org/advanced-administration/debug/debug-wordpress/\">Debugging in WordPress</a> for more information. (This message was added in version 6.7.0.) in /wordpress/wp-includes/functions.php on line 6121","level":"NOTICE","type":"PHP","test":"jspi-boot","plugin":"comments-counter-in-feed","details":"[30-May-2025 14:30:06 UTC] PHP Notice:  Function _load_textdomain_just_in_time was called <strong>incorrectly</strong>. Translation loading for the <code>cartnotify</code> domain was triggered too early. This is usually an indicator for some code in the plugin or theme running too early. Translations should be loaded at the <code>init</code> action or later. Please see <a href=\"https://developer.wordpress.org/advanced-administration/debug/debug-wordpress/\">Debugging in WordPress</a> for more information. (This message was added in version 6.7.0.) in /wordpress/wp-includes/functions.php on line 6121","log":"/logs/plugins/c/comments-counter-in-feed/error.json"}
,
{"message":"Function _load_textdomain_just_in_time was called <strong>incorrectly</strong>. Translation loading for the <code>cartnotify</code> domain was triggered too early. This is usually an indicator for some code in the plugin or theme running too early. Translations should be loaded at the <code>init</code> action or later. Please see <a href=\"https://developer.wordpress.org/advanced-administration/debug/debug-wordpress/\">Debugging in WordPress</a> for more information. (This message was added in version 6.7.0.) in /wordpress/wp-includes/functions.php on line 6121","level":"NOTICE","type":"PHP","test":"jspi-boot","plugin":"comments-counter-in-feed","details":"[30-May-2025 14:30:16 UTC] PHP Notice:  Function _load_textdomain_just_in_time was called <strong>incorrectly</strong>. Translation loading for the <code>cartnotify</code> domain was triggered too early. This is usually an indicator for some code in the plugin or theme running too early. Translations should be loaded at the <code>init</code> action or later. Please see <a href=\"https://developer.wordpress.org/advanced-administration/debug/debug-wordpress/\">Debugging in WordPress</a> for more information. (This message was added in version 6.7.0.) in /wordpress/wp-includes/functions.php on line 6121","log":"/logs/plugins/c/comments-counter-in-feed/error.json"}
,
{"message":"Function wpdb::prepare was called <strong>incorrectly</strong>. The query argument of wpdb::prepare() must have a placeholder. Please see <a href=\"https://developer.wordpress.org/advanced-administration/debug/debug-wordpress/\">Debugging in WordPress</a> for more information. (This message was added in version 3.9.0.) in /wordpress/wp-includes/functions.php on line 6121","level":"NOTICE","type":"PHP","test":"jspi-boot","plugin":"comments-counter-in-feed","details":"[30-May-2025 14:33:39 UTC] PHP Notice:  Function wpdb::prepare was called <strong>incorrectly</strong>. The query argument of wpdb::prepare() must have a placeholder. Please see <a href=\"https://developer.wordpress.org/advanced-administration/debug/debug-wordpress/\">Debugging in WordPress</a> for more information. (This message was added in version 3.9.0.) in /wordpress/wp-includes/functions.php on line 6121","log":"/logs/plugins/c/comments-counter-in-feed/error.json"}
,
{"message":"Function wpdb::prepare was called <strong>incorrectly</strong>. The query argument of wpdb::prepare() must have a placeholder. Please see <a href=\"https://developer.wordpress.org/advanced-administration/debug/debug-wordpress/\">Debugging in WordPress</a> for more information. (This message was added in version 3.9.0.) in /wordpress/wp-includes/functions.php on line 6121","level":"NOTICE","type":"PHP","test":"jspi-boot","plugin":"comments-counter-in-feed","details":"[30-May-2025 14:33:48 UTC] PHP Notice:  Function wpdb::prepare was called <strong>incorrectly</strong>. The query argument of wpdb::prepare() must have a placeholder. Please see <a href=\"https://developer.wordpress.org/advanced-administration/debug/debug-wordpress/\">Debugging in WordPress</a> for more information. (This message was added in version 3.9.0.) in /wordpress/wp-includes/functions.php on line 6121","log":"/logs/plugins/c/comments-counter-in-feed/error.json"}
,
{"message":"SQLSTATE[HY000]: General error: 1 no such table: wp_coding_blocks.","level":"FATAL","type":"SQL","test":"jspi-boot","plugin":"comments-counter-in-feed","details":"[30-May-2025 14:33:48 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>SELECT * FROM wp_coding_blocks</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: SELECT * FROM wp_coding_blocks | (no parameters)</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 no such table: wp_coding_blocks.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-includes/class-wpdb.php(3146): WP_SQLite_DB->query('SELECT * FROM w...')\n#2 /wordpress/wp-content/plugins/coding-blocks/includes/class-coding-blocks-activator.php(44): wpdb->get_results('SELECT * FROM w...')\n#3 /wordpress/wp-content/plugins/coding-blocks/coding-blocks.php(49): Coding_Blocks_Activator::activate()\n#4 /wordpress/wp-includes/class-wp-hook.php(324): activate_coding_blocks(false)\n#5 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)\n#6 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#7 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_coding...', false)\n#8 /internal/eval.php(20): activate_plugin('coding-blocks/c...')\n#9 {main}</pre>\n for query SELECT * FROM wp_coding_blocks made by activate_plugin, do_action('activate_coding-blocks/coding-blocks.php'), WP_Hook->do_action, WP_Hook->apply_filters, activate_coding_blocks, Coding_Blocks_Activator::activate, WP_SQLite_DB->query, WP_SQLite_DB->print_error","log":"/logs/plugins/c/comments-counter-in-feed/error.json"}
,
{"message":"SQLSTATE[HY000]: General error: 1 no such table: wp_coding_blocks.","level":"FATAL","type":"SQL","test":"jspi-boot","plugin":"comments-counter-in-feed","details":"[30-May-2025 14:33:48 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>SELECT loader FROM wp_coding_blocks</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: SELECT loader FROM wp_coding_blocks | (no parameters)</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 no such table: wp_coding_blocks.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-includes/class-wpdb.php(3146): WP_SQLite_DB->query('SELECT loader F...')\n#2 /wordpress/wp-content/plugins/coding-blocks/includes/class-coding-blocks-activator.php(50): wpdb->get_results('SELECT loader F...')\n#3 /wordpress/wp-content/plugins/coding-blocks/coding-blocks.php(49): Coding_Blocks_Activator::activate()\n#4 /wordpress/wp-includes/class-wp-hook.php(324): activate_coding_blocks(false)\n#5 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)\n#6 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#7 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_coding...', false)\n#8 /internal/eval.php(20): activate_plugin('coding-blocks/c...')\n#9 {main}</pre>\n for query SELECT loader FROM wp_coding_blocks made by activate_plugin, do_action('activate_coding-blocks/coding-blocks.php'), WP_Hook->do_action, WP_Hook->apply_filters, activate_coding_blocks, Coding_Blocks_Activator::activate, WP_SQLite_DB->query, WP_SQLite_DB->print_error","log":"/logs/plugins/c/comments-counter-in-feed/error.json"}
,
{"message":"SQLSTATE[HY000]: General error: 1 no such table: wp_coding_blocks_settings.","level":"FATAL","type":"SQL","test":"jspi-boot","plugin":"comments-counter-in-feed","details":"[30-May-2025 14:33:48 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>SELECT * FROM wp_coding_blocks_settings</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: SELECT * FROM wp_coding_blocks_settings | (no parameters)</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 no such table: wp_coding_blocks_settings.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-includes/class-wpdb.php(3146): WP_SQLite_DB->query('SELECT * FROM w...')\n#2 /wordpress/wp-content/plugins/coding-blocks/includes/class-coding-blocks-activator.php(79): wpdb->get_results('SELECT * FROM w...')\n#3 /wordpress/wp-content/plugins/coding-blocks/coding-blocks.php(49): Coding_Blocks_Activator::activate()\n#4 /wordpress/wp-includes/class-wp-hook.php(324): activate_coding_blocks(false)\n#5 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)\n#6 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#7 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_coding...', false)\n#8 /internal/eval.php(20): activate_plugin('coding-blocks/c...')\n#9 {main}</pre>\n for query SELECT * FROM wp_coding_blocks_settings made by activate_plugin, do_action('activate_coding-blocks/coding-blocks.php'), WP_Hook->do_action, WP_Hook->apply_filters, activate_coding_blocks, Coding_Blocks_Activator::activate, WP_SQLite_DB->query, WP_SQLite_DB->print_error","log":"/logs/plugins/c/comments-counter-in-feed/error.json"}
,
{"message":"Function wpdb::prepare was called <strong>incorrectly</strong>. The query argument of wpdb::prepare() must have a placeholder. Please see <a href=\"https://developer.wordpress.org/advanced-administration/debug/debug-wordpress/\">Debugging in WordPress</a> for more information. (This message was added in version 3.9.0.) in /wordpress/wp-includes/functions.php on line 6121","level":"NOTICE","type":"PHP","test":"jspi-boot","plugin":"comments-counter-in-feed","details":"[30-May-2025 14:33:58 UTC] PHP Notice:  Function wpdb::prepare was called <strong>incorrectly</strong>. The query argument of wpdb::prepare() must have a placeholder. Please see <a href=\"https://developer.wordpress.org/advanced-administration/debug/debug-wordpress/\">Debugging in WordPress</a> for more information. (This message was added in version 3.9.0.) in /wordpress/wp-includes/functions.php on line 6121","log":"/logs/plugins/c/comments-counter-in-feed/error.json"}
,
{"message":"SQLSTATE[HY000]: General error: 1 no such table: wp_coding_blocks.","level":"FATAL","type":"SQL","test":"jspi-boot","plugin":"comments-counter-in-feed","details":"[30-May-2025 14:33:58 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>SELECT * FROM wp_coding_blocks</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: SELECT * FROM wp_coding_blocks | (no parameters)</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 no such table: wp_coding_blocks.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-includes/class-wpdb.php(3146): WP_SQLite_DB->query('SELECT * FROM w...')\n#2 /wordpress/wp-content/plugins/coding-blocks/includes/class-coding-blocks-activator.php(44): wpdb->get_results('SELECT * FROM w...')\n#3 /wordpress/wp-content/plugins/coding-blocks/coding-blocks.php(49): Coding_Blocks_Activator::activate()\n#4 /wordpress/wp-includes/class-wp-hook.php(324): activate_coding_blocks(false)\n#5 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)\n#6 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#7 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_coding...', false)\n#8 /internal/eval.php(20): activate_plugin('coding-blocks/c...')\n#9 {main}</pre>\n for query SELECT * FROM wp_coding_blocks made by activate_plugin, do_action('activate_coding-blocks/coding-blocks.php'), WP_Hook->do_action, WP_Hook->apply_filters, activate_coding_blocks, Coding_Blocks_Activator::activate, WP_SQLite_DB->query, WP_SQLite_DB->print_error","log":"/logs/plugins/c/comments-counter-in-feed/error.json"}
,
{"message":"SQLSTATE[HY000]: General error: 1 no such table: wp_coding_blocks.","level":"FATAL","type":"SQL","test":"jspi-boot","plugin":"comments-counter-in-feed","details":"[30-May-2025 14:33:58 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>SELECT loader FROM wp_coding_blocks</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: SELECT loader FROM wp_coding_blocks | (no parameters)</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 no such table: wp_coding_blocks.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-includes/class-wpdb.php(3146): WP_SQLite_DB->query('SELECT loader F...')\n#2 /wordpress/wp-content/plugins/coding-blocks/includes/class-coding-blocks-activator.php(50): wpdb->get_results('SELECT loader F...')\n#3 /wordpress/wp-content/plugins/coding-blocks/coding-blocks.php(49): Coding_Blocks_Activator::activate()\n#4 /wordpress/wp-includes/class-wp-hook.php(324): activate_coding_blocks(false)\n#5 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)\n#6 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#7 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_coding...', false)\n#8 /internal/eval.php(20): activate_plugin('coding-blocks/c...')\n#9 {main}</pre>\n for query SELECT loader FROM wp_coding_blocks made by activate_plugin, do_action('activate_coding-blocks/coding-blocks.php'), WP_Hook->do_action, WP_Hook->apply_filters, activate_coding_blocks, Coding_Blocks_Activator::activate, WP_SQLite_DB->query, WP_SQLite_DB->print_error","log":"/logs/plugins/c/comments-counter-in-feed/error.json"}
,
{"message":"SQLSTATE[HY000]: General error: 1 no such table: wp_coding_blocks_settings.","level":"FATAL","type":"SQL","test":"jspi-boot","plugin":"comments-counter-in-feed","details":"[30-May-2025 14:33:58 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>SELECT * FROM wp_coding_blocks_settings</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: SELECT * FROM wp_coding_blocks_settings | (no parameters)</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 no such table: wp_coding_blocks_settings.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-includes/class-wpdb.php(3146): WP_SQLite_DB->query('SELECT * FROM w...')\n#2 /wordpress/wp-content/plugins/coding-blocks/includes/class-coding-blocks-activator.php(79): wpdb->get_results('SELECT * FROM w...')\n#3 /wordpress/wp-content/plugins/coding-blocks/coding-blocks.php(49): Coding_Blocks_Activator::activate()\n#4 /wordpress/wp-includes/class-wp-hook.php(324): activate_coding_blocks(false)\n#5 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)\n#6 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#7 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_coding...', false)\n#8 /internal/eval.php(20): activate_plugin('coding-blocks/c...')\n#9 {main}</pre>\n for query SELECT * FROM wp_coding_blocks_settings made by activate_plugin, do_action('activate_coding-blocks/coding-blocks.php'), WP_Hook->do_action, WP_Hook->apply_filters, activate_coding_blocks, Coding_Blocks_Activator::activate, WP_SQLite_DB->query, WP_SQLite_DB->print_error\n\n    at Object.onBind (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/index.js:237:15)\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async V (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/index.js:64:10)\n    at async ja (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/cli.js:8161:12) {\n  [cause]: Error: Error when executing the blueprint step #1 ({\"step\":\"installPlugin\",\"pluginData\":{\"resource\":\"wordpress.org/plugins\",\"slug\":\"comments-counter-in-feed\"},\"options\":{\"activate\":true}}) : Could not download \"https://downloads.wordpress.org/plugin/comments-counter-in-feed.latest-stable.zip\".\n  \t\t\t\tCheck if the URL is correct and the server is reachable.\n  \t\t\t\tIf it is reachable, the server might be blocking the request.\n  \t\t\t\tCheck the browser console and network tabs for more information.\n  \n  \t\t\t\t## Does the console show the error \"No 'Access-Control-Allow-Origin' header\"?\n  \n  \t\t\t\tThis means the server that hosts your file does not allow requests from other sites\n  \t\t\t\t(cross-origin requests, or CORS).\tYou need to move the asset to a server that allows\n  \t\t\t\tcross-origin file downloads. Learn more about CORS at\n  \t\t\t\thttps://developer.mozilla.org/en-US/docs/Web/HTTP/CORS.\n  \n  \t\t\t\tIf your file is on GitHub, load it from \"raw.githubusercontent.com\".\n  \t\t\t\tHere's how to do that:\n  \n  \t\t\t\t1. Start with the original GitHub URL of the file. For example:\n  \t\t\t\thttps://github.com/username/repository/blob/branch/filename.\n  \t\t\t\t2. Replace \"github.com\" with \"raw.githubusercontent.com\".\n  \t\t\t\t3. Remove the \"/blob/\" part of the URL.\n  \n  \t\t\t\tThe resulting URL should look like this:\n  \t\t\t\thttps://raw.githubusercontent.com/username/repository/branch/filename\n  \n  \t\t\t\tError:\n  \t\t\t\tError: Could not download \"https://downloads.wordpress.org/plugin/comments-counter-in-feed.latest-stable.zip\"\n      at Object.run (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/blueprints/index.js:12449:19)\n      at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n      at async ms (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/blueprints/index.js:12552:3)\n      at async Object.onBind (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/index.js:227:48)\n      at async V (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/index.js:64:10)\n      at async ja (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/cli.js:8161:12) {\n    [cause]: Error: Could not download \"https://downloads.wordpress.org/plugin/comments-counter-in-feed.latest-stable.zip\".\n    \t\t\t\tCheck if the URL is correct and the server is reachable.\n    \t\t\t\tIf it is reachable, the server might be blocking the request.\n    \t\t\t\tCheck the browser console and network tabs for more information.\n    \n    \t\t\t\t## Does the console show the error \"No 'Access-Control-Allow-Origin' header\"?\n    \n    \t\t\t\tThis means the server that hosts your file does not allow requests from other sites\n    \t\t\t\t(cross-origin requests, or CORS).\tYou need to move the asset to a server that allows\n    \t\t\t\tcross-origin file downloads. Learn more about CORS at\n    \t\t\t\thttps://developer.mozilla.org/en-US/docs/Web/HTTP/CORS.\n    \n    \t\t\t\tIf your file is on GitHub, load it from \"raw.githubusercontent.com\".\n    \t\t\t\tHere's how to do that:\n    \n    \t\t\t\t1. Start with the original GitHub URL of the file. For example:\n    \t\t\t\thttps://github.com/username/repository/blob/branch/filename.\n    \t\t\t\t2. Replace \"github.com\" with \"raw.githubusercontent.com\".\n    \t\t\t\t3. Remove the \"/blob/\" part of the URL.\n    \n    \t\t\t\tThe resulting URL should look like this:\n    \t\t\t\thttps://raw.githubusercontent.com/username/repository/branch/filename\n    \n    \t\t\t\tError:\n    \t\t\t\tError: Could not download \"https://downloads.wordpress.org/plugin/comments-counter-in-feed.latest-stable.zip\"\n        at Ct.resolve (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/blueprints/index.js:1472:13)\n        at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n        at async _.run (file:///home/runner/work/playground-tester/playground-tester/node_modules/@php-wasm/util/index.js:45:14)\n  }\n}\n\n","log":"/logs/plugins/c/comments-counter-in-feed/error.json"}
]
